visualizations/frontend/other/TTable.vue

```diff
@@ -380,6 +376,38 @@ export default {
     uniqueOriginalRows() {
       return _.uniq(this.internalRows.map((r) => r.originalRow));
     },
+    layerRows() {
+      if (typeof this.rows === "function")
+        return this.rows(this.$attrs?.["t-layer"]);
+      if (this.rows) return this.rows;
+      return [];
+    },
+  },
+  watch: {
+    loading() {
+      if (!this.loading) {
+        this.updateEndIndex(this.rowsPerPage);
+        this.updateStartIndex(0);
+        this.pagerResetFunction();
+        this.fetchTotalRows();
+      }
+      this.showSpinner = this.loading;
```

Comment:
and `showSpinner` could be converted into a `computed` value I guess, or you can rely on `this.loading` instead of creating a new state.

Reply:
I think at one point it didn't always match this.loading, let me check if that is still the case.

Reply:
No go, 😢 when the TTable fetches a new page of data "loading" isn't set to true so the spinner wouldn't show without the showSpinner state. We should probably revisit how fetching new pages is handled at some point to take advantage of your updates, and eliminate the need for showSpinner, but I think that is out of scope.

Reply:
Yep, I removed the watcher